testcomponent/features/updateMany.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Update Many Records
```

Comment:
There doesn't appear to be any sort of test (unit or component) driving:
1. the unhappy path functionality
2. upsert path (I could be wrong...)

Reply:
upsert isn't required for this use case as we don't want to insert the record if it doesn't exist.

Reply:
Resolved the missing unhappy path test

Reply:
Unhappy path testing added as a unit test in mongodb/collection_test.go